roles/edpm_bootstrap/defaults/main.yml

```diff
@@ -49,3 +49,9 @@ edpm_bootstrap_network_service: NetworkManager
 
 # String for SELinux state. One of: disabled, enforcing, permissive
 edpm_bootstrap_selinux_mode: enforcing
+
+# Swap management
+edpm_bootstrap_swap_size_megabytes: 4096
+edpm_bootstrap_swap_path: /swap
+edpm_bootstrap_swap_partition: false
```

Comment:
The var name does not represent well that this is a boolean flag to enable swap. Probably edpm_bootstrap_enable_swap (or swap_enabled) is better.